site/blog/OIS_Sessions_Blog

```diff
@@ -0,0 +1,53 @@
+---
+title: StarlingX sessions from Open Infrastructure Summit
+author: jeff-gowan
+date: 2019-05-22
+category: news
+---
+Did you miss out on seeing some of the StarlingX sessions at Open Infrastructure Summit? If so, don’t worry about it because this blog has you covered. 
```

Comment:
Add a hyperlink to the Summit videos: https://www.openstack.org/videos/summits/denver-2019

Reply:
Also please define OIS like Open Infrastructure Summit(OIS).

Reply:
Instead of abbreviating as OIS, can we just say Open Infrastructure Summit on the first mention and then the Summit on further mentions? It's not an abbreviation we have really been encouraging